app/helpers/serializer_helper.rb

```diff
@@ -13,4 +13,11 @@ def required_attributes(model, serializer)
 
     (serializer_attributes & model_attributes).map { |attr| "#{model.table_name}.#{attr}" }
   end
+
+  # Since Rails 4 has adjusted the way assets paths are handled, we have to access certain
+  # asset-based helpers like this, when outside of a view or controller context.
+  # See: https://stackoverflow.com/a/16609815
+  def image_path(path)
+    ActionController::Base.helpers.image_path(path)
+  end
```

Comment:
this is a global helper. globals are not good. can we move this to a service?

Reply:
> this is a global helper

Ah, right. I'll rethink this.

Reply:
The `SerializerHelper` seemed like I reasonable place to put it, but I forgot this will be available in all controllers and views. Yeah, that's awful. :see_no_evil:

Reply:
Extracted to a service :point_down: 

I wonder if a more general service that just gave access to helpers might be a better option? Like:
```ruby
class RailsHelpersService
  def self.helpers(path)
    ActionController::Base.helpers
  end
end
```
and then calling `RailsHelpersService.helpers.image_path('whatever.jpg')`

It's a lot more re-usable but it looks a bit weird....

Reply:
I think I prefer the yagni option here, a image path specific service instead of exposing all the helpers in a service.